### ISSUE TITLE
Orchid Accessibility Audit

### ISSUE THREAD

**Issue by erinchambers** [CDRH/orchid#316]
Create a temporary branch of the (now not-in-use) Ardhi dev site and remove all Orchid overrides. Switch the dev server to use the temp branch and have the WAE tool scan that. Then, switch it back to the dev branch.

# Version of site audited

**TEMP site url**: https://cdrhrailsdev.unl.edu/ardhi (this is an Orchid site no longer in use)
**TEMP branch**: [orchid_testing](https://github.com/CDRH/ardhi/tree/orchid_testing) (branch with all Orchid overrides removed, so that base Orchid code can be audited without customizations)

# Checklist

  - [x] Generate lists for this issue using the [Checklist of Products for ADA Title II](https://uofnebraska.sharepoint.com/:x:/r/sites/UNL-UniversityLibraries/DISC/cdrh/Shared%20Documents/CDRH%20Projects/Project%20Folders/Accessibility%20Audits/2025-2026%20ADA%20Title%20II%20Project/Planning%20Documents/Checklist%20of%20Products%20for%20ADA%20Title%20II%20Project.xlsx?d=w52aafefaa1a841b09bae916225c22cbb&csf=1&web=1&e=mvYOdi) spreadsheet
  - [x] Identify sections and URLs that duplicate search/browse for each section
  - [x] Identify representative items/pages from each section
  - [x] Identify any other unique, non-item pages within each section
  - [x] Compile in spreadsheet (linked above)

## `robots.txt`
  - [x] Upload `robots.txt`. Include:
    - /browse/
    - /search/
    - for each section: search/browse/other views with paginated results

### Contents of `robots.txt` for this site:

```
User-agent: *
Disallow: /browse/
Disallow: /search/
```

## Site crawl

- [x] Scan site using [WAE tool](https://wae.unl.edu/)

```
https://cdrhrailsdev.unl.edu/ardhi
```

## Page scans

Pages scanned individually using axeDevTools web browser extension:

## Manual testing

Pages tested manually for things that automated scans cannot detect:

  - [x] Home page
  - [x] Primary "static" pages
    - [ ] 

## Delete robots.txt

- [x] Remove robots.txt from server
